assets/src/application/ui/calendars/dateDisplay/CalendarDateSwitcher/index.tsx

```diff
@@ -1,26 +1,30 @@
 import React from 'react';
-import classNames from 'classnames';
 import { parseISO } from 'date-fns';
 import { __ } from '@wordpress/i18n';
 
+import { BiggieCalendarDate, CalendarDateRange } from '../';
+import { CalendarDateSwitcherProps } from './types';
 import { DisplayStartOrEndDate } from '@sharedServices/filterState';
 import { PLUS_ONE_MONTH, PLUS_TWO_MONTHS } from '@sharedConstants/defaultDates';
-import { BiggieCalendarDate, CalendarDateRange } from '../';
+import switchTenseForDate from '@sharedServices/utils/switchTenseForDate';
 
-interface Props {
-	className?: string;
-	displayDate: DisplayStartOrEndDate;
-	endDate: string;
-	footerText?: string;
-	headerText?: string;
-	showDate?: boolean;
-	startDate: string;
-}
+const headerFuture = 'sale starts';
+const headerPast = 'sale started';
+const footerFuture = 'sale ends';
+const footerPast = 'sale ended';
 
-const CalendarDateSwitcher: React.FC<Props> = React.memo(
-	({ className, displayDate = DisplayStartOrEndDate.start, footerText, headerText, ...props }) => {
+const CalendarDateSwitcher: React.FC<CalendarDateSwitcherProps> = React.memo(
```

Comment:
@tn3rb should we pass ``headerFuture`,  `headerPast`, `footerFuture`, `footerPast`, such that we can control labels in certain situations(maybe) ?